Subject: Add --disable-silent-rules support in mklove
Forwarded: https://github.com/edenhill/mklove/pull/13

diff --git a/mklove/modules/configure.builtin b/mklove/modules/configure.builtin
index 55d478d0a004..293e084d69c9 100644
--- a/mklove/modules/configure.builtin
+++ b/mklove/modules/configure.builtin
@@ -46,4 +46,5 @@ mkl_toggle_option "Compatibility" "mk:MKL_MAINT_MODE" "--enable-maintainer-mode"
 mkl_option "Configure tool" "mk:PROGRAM_PREFIX" "--program-prefix=PFX" "Program prefix"
 
 mkl_option "Compatibility" "mk:DISABL_DEP_TRACK" "--disable-dependency-tracking" "Disable dependency tracking (no-op)"
+mkl_option "Compatibility" "mk:DISABL_SILENT_RULES" "--disable-silent-rules" "Disable silent rules (no-op)"
 
